projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/select-docs/select-docs.component.html

```diff
@@ -785,4 +785,48 @@ <h4 class="go-heading-4 go-heading--underlined">Code</h4>
 
     </div>
   </go-card>
+  <go-card id="form-select-template" class="go-column go-column--100">
+    <ng-container go-card-header>
+      <h2 class="go-heading-2 go-heading--no-wrap">Component Virtual scroll</h2>
+      <go-copy cardId="form-select-template" goCopyCardLink></go-copy>
+    </ng-container>
+    <div class="go-container" go-card-content>
+      <div class="go-column go-column--100">
+        <p class="go-body-copy go-body-copy--no-margin">
+          At times there might be loading many items but only limited of them are rendered in the DOM. <code
+            class="code-block--inline">scrollToEnd</code> allows users to load a set of data when user scrolls till the
+          end in the DOM. <br />
+          Scroll event will work only after enabling <code class="code-block--inline">virtualScroll</code> to true.
+        </p>
```

Comment:
```html
<p class="go-body-copy">
  Sometimes the data that needs to be rendered for the options of the select would cause performance and user 
  experience issues. In this case, virtual scroll should be implemented.
</p>

<p class="go-body-copy">
  To do so, implement the <code class="code-block--inline">scrollToEnd</code> method on the select. When 
  the user scrolls to the end of the current available options, the event will fire which will allow the implementor to 
  fetch more options and add them to the array of items. This event will only fire after setting <code class="code-block--inline">virtualScroll</code> to true on the select component.
</p>

<p class="go-body-copy go-body-copy--no-margin">
  For more information on this, check the
  <a
    href="https://ng-select.github.io/ng-select#/virtual-scroll"
    class="go-link">
      ng-select documentation
  </a>
  for implementing virtual scroll.
</p>
```

Reply:
I went ahead and rewrote this, just to clean up the wording a bit. No worries there, but probably would be easiest if you copied over what I wrote here.